src/java/com/zimbra/graphql/repositories/impl/ZXMLAccountRepository.java

```diff
@@ -89,12 +98,43 @@ public ZXMLAccountRepository() {
      * @param modifyPrefsHandler The pref mutation handler
      */
     public ZXMLAccountRepository(GetAccountInfo accountInfoHandler, EndSession endSessionHandler,
-        GetPrefs prefsHandler, ModifyPrefs modifyPrefsHandler) {
+        GetPrefs prefsHandler, ModifyPrefs modifyPrefsHandler, GetInfo infoHandler) {
         super();
         this.accountInfoHandler = accountInfoHandler;
         this.endSessionHandler = endSessionHandler;
         this.prefsHandler = prefsHandler;
         this.modifyPrefsHandler = modifyPrefsHandler;
+        this.infoHandler = infoHandler;
+    }
+
+    /**
+     * Retrieves info.
+     *
+     * @param rctxt The request context
+     * @param sections Comma separated list of sections to return information about
+     * @param rights Comma separated list of rights to return information about
+     * @return The GetInfoResponse as per the request
+     * @throws ServiceException If there are issues executing the document
+     */
+    public GetInfoResponse infoGet(RequestContext rctxt, String sections, String rights) throws ServiceException{
+        final ZimbraSoapContext zsc = GQLAuthUtilities.getZimbraSoapContext(rctxt);
+        GetInfoRequest request = new GetInfoRequest();
+        if (!StringUtil.isNullOrEmpty(sections)) {
```

Comment:
This may be true, and a good idea - but it is better to keep it consistent with this project for now and make the sweeping change across all files in one shot later. That way if we never come back and do it to every other file - we are still consistent within the project.